docker/config/local_dev.env

```diff
@@ -27,7 +27,7 @@ CRASHSTORAGE_BUCKET_NAME=antennabucket
 # Pub/Sub things for local development
 CRASHPUBLISH_SERVICE_ACCOUNT_FILE=
 CRASHPUBLISH_PROJECT_ID=local_dev_socorro
-CRASHPUBLISH_TOPIC_NAME=local_dev_socorro_normal
+CRASHPUBLISH_TOPIC_NAME=local_dev_socorro_standard
```

Comment:
The name we're using in the documentation is "standard". Before we switched to pubsub, it was sometimes called "standard" and sometimes called "normal". Figure it's easier if I remove the "normal" case everywhere so it's easier to compare pubsub to sqs.